platform-configuration/src/main/java/org/codice/ddf/configuration/ConfigurationManager.java

```diff
@@ -271,18 +313,16 @@ public void setConfigurationAdmin(ConfigurationAdmin configurationAdmin) {
 
     /**
      * Retrieves the value of an OSGi bundle's configuration property
-     * 
-     * @param servicePid
-     *            PID for an OSGi bundle
-     * @param propertyName
-     *            name of the bundle's configuration property to get a value for
-     * 
+     *
+     * @param servicePid   PID for an OSGi bundle
+     * @param propertyName name of the bundle's configuration property to get a value for
      * @return the value of the specified bundle's configuration property
      */
     public String getConfigurationValue(String servicePid, String propertyName) {
         String methodName = "getConfigurationValue";
-        logger.debug("ENTERING: " + methodName + "   servicePid = " + servicePid
-                + ",  propertyName = " + propertyName);
+        logger.debug(
```

Comment:
Really this class should just use parameterized logging. It's a much cleaner way to achieve the same goal.